source/jobs/sync-jobs.txt

```diff
@@ -40,6 +40,19 @@ Recoverability
 
 .. include:: /includes/fact-cdc-recoverable.rst
 
+Behavior
+--------
+
+Sync jobs are non-impodent by default. Relational Migrator inserts new 
+documents and embeded arrays each time a job is run, with the option to 
+delete each existing collection at the start of the sync dialogue.
+
+You can enable impodent inserts for documents and embedded arrays to 
```

Comment:
Consider rewrite in form: "To XXX, set YYY."

Reply:
Good suggestion! I also simplified the sentence and moved a couple of things around, while providing the same info.